subsys/pm/policy/policy_latency.c

```diff
@@ -16,35 +16,34 @@ static struct k_spinlock latency_lock;
 /** List of maximum latency requests. */
 static sys_slist_t latency_reqs;
 /** Maximum CPU latency in us */
-static int32_t max_latency_us = SYS_FOREVER_US;
+static uint32_t max_latency_us = SYS_FOREVER_US;
```

Comment:
This feels like a compiler warning waiting to happen, same with all the other assignments of `-1` to a `uint32_t` in this PR.